src/pages/Login/LoginEmail.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import GoogleAnalytics from 'react-ga'
+import Input from '@santiment-network/ui/Input'
+import Button from '@santiment-network/ui/Button'
+import Icon from '@santiment-network/ui/Icon'
+import gql from 'graphql-tag'
```

Comment:
maybe combine `Input`, `Button` and `Icon` in one import?

Reply:
_TL;DR;_ Different import statements are better for webpack

**Webpack** has poor tree-shaking (almost none) in case of importing multiple exports from module.